### PR TITLE
Spelling Corrections

### PR DESCRIPTION
**Pull request by RobLui**
I am targeting this branch, because Spelling corrections.

## Subject

Spelling corrections.

### CODE REVIEW

File: docs/reference/api.rst [hunk 1]
```diff
@@ -29,4 +29,4 @@ client only sent partial data to update an entity.
         // ...
     }
 
-The call need to be done before the ``bind`` function as you cannot manipulate the form after the binding.
+The call needs to be done before the ``bind`` function as you cannot manipulate the form after the binding.
```
Comment (kunicmarko20) [inline docs/reference/api.rst]: Please comply with the sf docs standards:

> [Each line should break approximately after the first word that crosses the 72nd character (so most lines end up being 72-78 characters);](http://symfony.com/doc/current/contributing/documentation/standards.html)

Comment (RobLui) [reply docs/reference/api.rst]: sure :-)

**Comment by jordisala1991** [comment]
Ping @RobLui

File: docs/reference/api.rst (hunk 1 above)
Comment (greg0ire) [reply docs/reference/api.rst]: Ping?

File: docs/reference/form_types.rst [hunk 2]
```diff
@@ -38,8 +38,10 @@ The service definition should contain the following arguments:
 
 .. warning::
 
-    ``DoctrineORMSerializationType`` cannot be used directly with Symfony3.0, you need to extends the class ``BaseDoctrineORMSerializationType``
-    with an empty class to have an unique FQCN.
+    ``DoctrineORMSerializationType`` cannot be used directly with
```
Comment (kunicmarko20) [inline docs/reference/form_types.rst]: This is deprecated and removed on master, maybe remove it from docs?

Comment (greg0ire) [reply docs/reference/form_types.rst]: Yes, but in another PR

**Pull request merged**